examples/summarization/bart/evaluate_cnn.py

```diff
@@ -63,8 +105,14 @@ def run_generate():
         "--bs", type=int, default=8, required=False, help="batch size: how many to summarize at a time",
     )
     args = parser.parse_args()
-    examples = [" " + x.rstrip() for x in open(args.source_path).readlines()]
+    examples = [" " + x.rstrip() if "t5" in args.model_name else x.rstrip() for x in open(args.input_path).readlines()]
```

Comment:
is this necesarry @patrickvonplaten ?

Reply:
Yeah, for the current way we preprocess the  data it is. But when we switch to `nlp` this line is not necessary anymore